api/api/settings.py

```diff
@@ -214,6 +214,17 @@
 WATCHDOG_SLAVES = os.environ.get('DESECSTACK_WATCHDOG_SLAVES', '').split()
 WATCHDOG_WINDOW_SEC = 600
 
+# Prometheus
```

Comment:
Add "TODO" so that this can hopefully be resolved one day

Reply:
done